scripts/runtestsuite/runtestsuite.py

```diff
@@ -522,25 +587,20 @@ def main():
     utils.print_runtime_details("runtestsuite", VERSION, os.path.realpath(__file__), CURRENT_TIME)
     print_configuration(args.tool_path, appleseed_args)
 
-    start_time = datetime.datetime.now()
-    rendered_scene_count, passing_scene_count = render_test_scenes(script_directory, args)
-    end_time = datetime.datetime.now()
-
-    success = 100.0 * passing_scene_count / rendered_scene_count if rendered_scene_count > 0 else 0.0
+    failures, rendered_scenes, success_rate, total_time = render_test_scenes(script_directory, args)
```

Comment:
Some times, when you end up passing multiple argument everytime together, it's much easier to create an object.

Like `class TestSuiteRunnerResult`

Reply:
Yup, I like it.

Reply:
I'm thinking maybe something shorter (I notice `Logger` and not `TestSuiteLogger` for instance). How about `TestResult` or just `Results`?

Reply:
Logger is something very common and speek for itself.

Results here is really only for the tests and doesn't speek for itself. You can't really use it ouside